[0m10:23:49.519155 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105c08aa0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x1058044d0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105804440>]}


============================== 10:23:49.521221 | 0a7d7ed0-dcde-4765-a8a7-967b89df74e5 ==============================
[0m10:23:49.521221 [info ] [MainThread]: Running with dbt=1.8.7
[0m10:23:49.521474 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/Users/sundari/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt init demo_dbt', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m10:23:49.528906 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse dict {'dir': PosixPath('/Users/sundari/.dbt')}
[0m10:23:49.529094 [info ] [MainThread]: Creating dbt configuration folder at 
[0m10:23:49.529833 [debug] [MainThread]: Starter project path: /Library/Frameworks/Python.framework/Versions/3.12/lib/python3.12/site-packages/dbt/include/starter_project
[0m10:23:49.534877 [info ] [MainThread]: 
Your new dbt project "demo_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m10:23:49.535070 [info ] [MainThread]: Setting up your profile.
[0m10:23:49.535370 [error] [MainThread]: Encountered an error:
Runtime Error
  No adapters available. Learn how to install an adapter by going to https://docs.getdbt.com/docs/connect-adapters#install-using-the-cli
[0m10:23:49.536831 [debug] [MainThread]: Resource report: {"command_name": "init", "command_wall_clock_time": 0.044974376, "process_user_time": 0.444529, "process_kernel_time": 0.0779, "process_mem_max_rss": "92733440", "command_success": false, "process_in_blocks": "0", "process_out_blocks": "0"}
[0m10:23:49.537122 [debug] [MainThread]: Command `dbt init` failed at 10:23:49.537073 after 0.05 seconds
[0m10:23:49.537299 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105f85f70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105efae40>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x105bc3e00>]}
[0m10:23:49.537449 [debug] [MainThread]: Flushing usage events
